_conferences/devfest-bucharest-2019.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "DevFest"
```

Comment:
We usually call these "GDG DevFest"

Reply:
That's a good point. I've made the update.